src/components/ChatUI.tsx

```diff
@@ -198,6 +190,11 @@ const ChatUI: React.FC<ChatUIProps & { embedded?: boolean }> = ({
 
   const [isLoading, setIsLoading] = useState(false);
   const [isPortForwardRunning, setIsPortForwardRunning] = useState(false);
+
+  const [mcpServers, setMcpServers] = useState<MCPServer[]>([]);
+  const [mcpManagerOpen, setMcpManagerOpen] = useState(false);
+  const [mcpEnabled, setMcpEnabled] = useState(true);
```

Comment:
nit: setMcpEnabled was never used. Maybe it doesn't have to be a component state